gradio/chat_interface.py

```diff
@@ -362,7 +407,49 @@ def _setup_example_messages(
                 examples_messages.append(example_message)
         return examples_messages
 
+    def _generate_chat_title(self, conversation: list[MessageDict]) -> str:
```

Comment:
feels overly complicated, would prefer to just handle this automatically within the dataset component and add [text-overflow: ellipses](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow) support to it's values.

Reply:
I tried this approach but I couldn’t get it to work because the textbox would never overflow inside of the table